crates/esthri/Cargo.toml

```diff
@@ -44,6 +44,8 @@ tokio-util = { version = "0.7", features = ["io"] }
 walkdir = "2"
 strum = "0.24.1"
 strum_macros = "0.24.3"
+reqwest = { version="0.11.12", features=["stream", "json"] }
```

Comment:
could we slap this into test dependency,
presigned urls dont need https i believe

so u can probably also do
`default-features = false`

Reply:
reqwest is used in the presigned url client code - I could make this a specific feature maybe `presigned_url_client`.